tests/build_/test_build_request.py

```diff
@@ -960,16 +950,11 @@ def test_render_prod_request_requires_newer(self):
         with pytest.raises(OsbsValidationException):
             build_request.render()
 
-    @pytest.mark.parametrize('registry_api_versions', [
-        ['v1'],
-        ['v1', 'v2'],
-        ['v2'],
-    ])
     @pytest.mark.parametrize('platform', [None, 'x86_64'])
     @pytest.mark.parametrize('arrangement_version',
                              list(range(3, TEST_ARRANGEMENT_VERSION + 1)))
     @pytest.mark.parametrize('scratch', [False, True])
-    def test_render_prod_request_v1_v2(self, registry_api_versions, platform, arrangement_version,
+    def test_render_prod_request_v1_v2(self, platform, arrangement_version,
```

Comment:
change name of test to : test_render_prod_request_v2

Reply:
Thanks, fixed.

Reply:
Changed this back actually, to include a test that v1 is disallowed.